### PR TITLE
Update the appendix with FH HPC info

### PR DESCRIPTION
**Pull request by caalo**
Moved relevant content from https://github.com/fhdsl/WDL_Workflows_Guide/pull/25 to here.

(My RStudio reformatted everything a little bit, hence all the changes.)

### CODE REVIEW

File: 09-appendix-backends.Rmd [hunk 1]
```diff
@@ -36,65 +40,83 @@ If you are using the Dockstore CLI, [follow these instructions](https://docs.doc
...
-* The `memory` and `gpu` runtime attributes are ignored, but `cpu` works as expected
+-   [PROOF](https://proof.fredhutch.org/) is a Shiny frontend that launches Cromwell Server and lets you manage your jobs from a GUI. User guide can be found [here](https://sciwiki.fredhutch.org/dasldemos/proof-how-to/).
+
+-   The Fred Hutch HPC supports the use of multiple JSON files going into the same workflow
+
+-   The Fred Hutch HPC supports the use of Docker, so the `docker` runtime attribute works as expected
+
+-   `memory` and `gpu` runtime attributes are ignored, but `cpu` works as expected. Here is [some information](https://sciwiki.fredhutch.org/scicomputing/compute_jobs/#memory) on approximating memory use.
+
+-   `walltime` attribute is a string ("HH:MM:SS") that specifies how much time you want to request for the task. Can also specify \>1 day, e.g. "1-12:00:00" is 1 day+12 hours.
```
Comment (aofarrel) [inline 09-appendix-backends.Rmd:95]: I'd recommend you elaborate on whether this is a hard or soft limit

File: 09-appendix-backends.Rmd [hunk 2]
```diff
@@ -36,65 +40,83 @@ If you are using the Dockstore CLI, [follow these instructions](https://docs.doc
...
 
-* The Fred Hutch HPC supports the use of multiple JSON files going into the same workflow
+-   `partition` attribute specifies the [cluster partition](https://sciwiki.fredhutch.org/compdemos/gizmo_partition_index/)to use. The default is "campus-new".
+
+-   `modules` attribute is a a space-separated list of the [environment modules](https://sciwiki.fredhutch.org/scicomputing/compute_environments/) you'd like to load (in that order) prior to running the task. For example, "modules: "GATK/4.2.6.1-GCCcore-11.2.0 SAMtools/1.16.1-GCC-11.2.0", the GATK module will be loaded first, followed by the SAMtools module.
+
+-   `dockerSL` attribute allows users to use Docker and softlinks only to specific locations on the FH HPC Scratch disk. It is helpful when working with very large files. An example is: "ubuntu:latest". Just like the `docker` attribute, you should only specify this if you want the task to run in a container (otherwise the default will be a non-containerized job).
```
Comment (aofarrel) [inline 09-appendix-backends.Rmd:101]: This says "specific locations" but the example given looks like just a Docker string, not a specific location. Is this intentional?

Or do you mean that setting this variable means you can access specific locations as specified in the body of a workflow task?

**Review by aofarrel — CHANGES_REQUESTED** [review]
Just need a lil more information on two new additions. Everything else looks great!

File: 09-appendix-backends.Rmd (hunk 1 above)
Comment (caalo) [reply 09-appendix-backends.Rmd:95]: Not completely sure whether its a hard or soft limit - linked out to more info on walltime.

File: 09-appendix-backends.Rmd (hunk 2 above)
Comment (caalo) [reply 09-appendix-backends.Rmd:101]: No idea. Got it from @sitapriyamoorthi's writing - can you help us out?

I can't find this attribute on FH HPC's official documentation, so I'm a bit tempted to leave this out.

Comment (caalo) [reply 09-appendix-backends.Rmd]: Put it in issue https://github.com/fhdsl/WDL_Workflows_Guide/issues/49

**Pull request merged**